tinygrad/runtime/ops_gpu.py

```diff
@@ -9,8 +9,16 @@
 # see test/external/external_osx_profiling.py to determine this ratio. it's in like GPU clocks or something
 OSX_TIMING_RATIO = (125/3) if OSX else 1.0
 
+@functools.lru_cache(maxsize=1)
+def _get_errors():
+  ed = {getattr(cl, k): k for k in filter(lambda e: e.startswith("CL_"), dir(cl)) if getattr(cl, k) <=0}
+  print(ed)
+  return ed
+_ = _get_errors()
```

Comment:
something like `cl_errors = {getattr(cl, k): k for k in filter(lambda e: e.startswith("CL_"), dir(cl)) if getattr(cl, k) <= 0}` is enough.

Reply:
The print is extra debug I missed.

For the call of get errors, that is how I interpreted @geohot comment "do at init", as if I moved the method to a any of the `CL*` classes it will become a messy PR... so the only "init" left is the one done while loading the file so that the dictionary can be created when loaded.